project1/server/file_handler/my_file_handler.cpp

```diff
@@ -1,71 +1,47 @@
-//
-// Created by jake on 1/25/21.
-//
-
 #include "my_file_handler.h"
 #include <bits/stdc++.h>
 
 namespace server::file_handler{
-  bool file_handler::MyFileHandler::Delete(const std::string &filename) {
-
-    //convert filename to char*
-    int n = filename.length();
-    char fileName[n+1];
-    strcpy(fileName, filename.c_str());
+  bool MyFileHandler::Delete(const std::string &filename) {
 
-    //delete file
-    return remove(fileName);
+    return remove(filename.c_str());
+    
   }//Delete
-  bool file_handler::MyFileHandler::Put(const std::string &filename, const std::vector<uint8_t> &contents) {
+  bool MyFileHandler::Put(const std::string &filename, const std::vector<uint8_t> &contents) {
 
     //convert contents to string format
     std::string str_contents(contents.begin(), contents.end());
 
     //open file
-    try {this->os.open(filename);}
-    catch (std::exception &e) {
-      this->os.close();
-      return false;
-    }
+    this->os.open(filename);
 
     //write contents to file
-    try{this->os << str_contents;}
-    catch (std::exception &e){
-      this->os.close();
-      return false;
-    }
+    this->os << str_contents;
 
     //close file
     this->os.close();
+    
     return true;
   }//Put
-  bool file_handler::MyFileHandler::MakeDir(const std::string &name) {
-    //convert directory name to char*
-    int n = name.length();
-    char dir[n+1];
-    strcpy(dir, name.c_str());
-
-    if (mkdir(dir,0777)==-1) return false;
-    else return true;
+  bool MyFileHandler::MakeDir(const std::string &name) {
+    
+    return mkdir(name.c_str(),0777)==-1;
```

Comment:
Um... should this be `!= -1`? Or maybe `== 0` would be more readable?

Reply:
Yeah, `== 0 `, it should be.